src/native/minipal/memory.h

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifndef MINIPAL_MEMORY_H
+#define MINIPAL_MEMORY_H
+
+#include <stdlib.h>
+
+#ifdef __cplusplus
+    extern "C"
+    {
+#endif // __cplusplus
+
+#ifndef HOST_WINDOWS
+inline void* CoTaskMemAlloc(size_t cb)
```

Comment:
cc @lambdageek, who originally suggested adding a `minipal_` prefix to all minipal APIs for clarity on their source. Up to now, all minipal APIs have consistently followed this naming convention.

Reply:
Should this be called minicom or something to make it clear that it is not minipal proper?

Reply:
I can move this into the `minipal/com` directory where I initially had it to make that more clear.

Reply:
Yeah I'll do that and move this back to that subfolder.

Reply:
Looks like the subfolder work is in the DNMD PR itself, so that would be more complicated than necessary here. I'll just prefix it with `minicom_` and go back to having an implementation on each platform.

Reply:
I meant that the include file should be called minicom. I did not mean to prefix the COM defs with it. We would not be able to apply the prefix for whatever is used in MIDL generated files that will lead to inconsistencies.

Reply:
Okay I'll move it into a subdirectory instead and set up that like I was planning on doing with DNMD.